### DISCUSSION TITLE
Login for all Twilio apps

### DISCUSSION THREAD

**Discussion by bchicca** [twilio-labs/paste#3229] [General] [answered]
Hello, 
Can you please confirm whether [this is the login](https://www.figma.com/file/pcInrjkl9tndHyxoOEXj5p/Log-In-Experience?type=design&node-id=14018-106755&t=8kxFl1fvGB90L9Mw-4) we should use for all Twilio apps?

We are currently using this but we're not sure where it's coming from: 

![image (3)](https://github.com/twilio-labs/paste/assets/114075970/19116211-3bfb-447f-b92d-a420ce5677c2)

Thanks!

**Comment by loreina** [comment, marked as answer]
Hey @bchicca,

Paste doesn't have a login pattern–I reached out to @amyodea who works on Identity and she'll follow up with you over Slack. Thanks!
